website_portal_v10/README.rst

```diff
@@ -82,6 +82,10 @@ welcomed feedback.
 Credits
 =======
 
+**This module is a backport from Odoo SA and as such, it is not included in the
+OCA CLA. That means we do not have a copy of the copyright on it like all
+other OCA modules.**
+
```

Comment:
I'm not board- but in the Contributing guidelines in the Backporting section, this is stated - https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#backporting-odoo-modules